examples/run.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2020 Fredy Ramirez <https://formateli.com>
+# For license see LICENSE
+"""
+run.py
+Show all examples located in this example folder.
+Main window show a button for each example.
+"""
+import sys, os
+import subprocess
+import tkinter as tk
+from tkinter import ttk
+
+_DIR = os.path.dirname(os.path.realpath(__file__))
+_DIR = os.path.normpath(os.path.join(_DIR, '..'))
+
+EXAMPLE_ENV = os.environ.copy()
+if not 'PYTHONPATH' in EXAMPLE_ENV:
+    EXAMPLE_ENV["PYTHONPATH"] = _DIR
+else:
+    EXAMPLE_ENV["PYTHONPATH"] = _DIR + os.pathsep + EXAMPLE_ENV["PYTHONPATH"]
+
+
+class _SampleButton:
+    def __init__(self, root, text, col, row):
+        self.root = root
+        self.btn = ttk.Button(root, text=text, command=self.run_example)
+        self.btn.grid(row=row, column=col, sticky="nsew") 
+
+    def run_example(self, event=None):
+        try:
+            subprocess.run(
+                [
+                    'python',
+                    'example_' + self.btn['text'] + '.py'
+                ],
+                env=EXAMPLE_ENV
+            )
+        except Exception as e:
+            print(e)
```

Comment:
I do not see a particular need to catch the exception here. The traceback printed normally will be far more useful and the Tkinter mainloop will not be cancelled as the exception is caught within it.

Reply:
Done